crates/solvers-dto/src/auction.rs

```diff
@@ -728,13 +728,17 @@ pub struct I32(String);
 /// represent the owner address and bytes [52, 56) represent the order's
 /// `validTo` field.
 #[derive(ToSchema)]
-#[schema(example = "0x30cff40d9f60caa68a37f0ee73253ad")]
+#[schema(
+    example = "0x30cff40d9f60caa68a37f0ee73253ad6ad72b45580c945fe3ab67596476937197854163b1b0d24e77dca702b97b5cc33e0f83dcb626122a6"
+)]
 #[allow(dead_code)]
 pub struct OrderUid(String);
 
 /// Signature bytes.
 #[derive(ToSchema)]
-#[schema(example = "0x0000000000000000000000000000000")]
+#[schema(
+    example = "0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
Can we use proper examples not default values?

Reply:
Sure, will address it in the main PR.